extensions/widgets/spinner/spinner.lcb

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright (C) 2015-2016 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+/**
+This widget is a spinner or activity indicator. Spinners provide visual
+feedback to users use when performing an activity for an unknown
+duration such as processing a large amount of data or presenting a
+complex user interface.
+*/
+
+-- declaring extension as widget, followed by identifier
+widget com.livecode.widget.spinner
```

Comment:
What do you think about calling this something like `dots_spinner`? We may wish to add other spinner variants in the future, such as an Android-style ring spinner.